### PR TITLE
[IMP] XML and language file improvments

### PR DESCRIPTION
**Pull request by zero-24**
Dear @Spudley with this PR I suggest some XML and language file improvments.

This includes:
- removal of unused language strings
- cleanup the main xml file
- remove the index.html (as they are not an requirement any more)
- change the language import to be local in the plugin
- add an script.php to enforce the joomla and php requirements on install and update as well as delete the outdated language files
- rename the XML Description language string from `PLG_USER_HIBP_DESCRIPTION` to `PLG_USER_HIBP_XML_DESCRIPTION`

### CODE REVIEW

File: hibp.xml [hunk 1]
```diff
@@ -1,26 +1,31 @@
 <?xml version="1.0" encoding="utf-8"?>
-<extension version="3.8" type="plugin" group="user" client="site" order="100" method="upgrade">
-	<name>User - HIBP</name>
+<extension version="3.8" type="plugin" group="user" client="site" method="upgrade">
+	<name>plg_user_hibp</name>
```
Comment (Spudley) [inline hibp.xml:3]: Shouldn't the language reference in the name element be upper case?

**Comment by Spudley** [comment]
I've made one small comment on this which I'd like to clarify. Apart from that I'll be happy to merge.

File: hibp.xml (hunk 1 above)
Comment (zero-24) [reply hibp.xml:3]: Well we could argue this is the name of the plugin that is also a language string :-D I personally more like to write it in lowercase at this place and iirc it is also done so in the core. But as this is nothing technical relevant i'm happy to write it in uppercase too, just let me know what you think would fit best
 :-)

**Comment by Spudley** [comment]
Merged in. All is good. 😄

**Pull request merged**